Dangerfile.swift

```diff
@@ -0,0 +1,12 @@
+import Danger 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 1;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)